### ISSUE TITLE
deploy.py fails with --repo option for a local relative directory

### ISSUE THREAD

**Issue by mdklatt** [mdklatt/cookiecutter-python-app#1]
Trying to deploy from the project directory fails, i.e. `deploy.py --repo=.`.

deploy.py:83:

```
with tmpdir():
    clone = "git clone {:s} {:s}".format(args.repo, _NAME)
    check_call(split(clone))
```

**Comment by mdklatt** [comment]
Relative paths need to be resolved before changing the working directory to the tmp directory.

**Comment by mdklatt** [comment]
This works if `args.repo` is a local file path, but if it's a Git URL it will be treated like a local path.

```
repo = abspath(args.repo)
with tmpdir():
    clone = "git clone {:s} {:s}".format(repo, _NAME)
    check_call(split(clone))
```

**Comment by mdklatt** [comment]
Issue resolved by commit 6950bc2.
